tracy-client/src/lib.rs

```diff
@@ -34,18 +37,33 @@ macro_rules! span {
         $crate::span!($name, 62)
     };
     ($name:expr, $callstack_depth:expr) => {{
+        use std::ffi::CString;
```

Comment:
Should re-export `CString` from the `$crate` as well. I would prefer a `#[doc(hidden)] pub mod internal` with most of the the internal stuff hidden in there over hiding each separate item.

Reply:
Is renaming std a thing? `std::any::type_name` wasn't reexported previously.

Reply:
I kinda did that a couple of times for codebases that needed to make libstd support conditional. But I agree the `CString`/`type_name` reexport doesn't need to happen now or here.